plugins/datadog-standalone/datadog-standalone.go

```diff
@@ -83,6 +83,8 @@ var (
 			}
 			if result.Status == "ok" {
 				log.Info("Successfully transmitted metrics")
+			} else if result.Status != nil {
```

Comment:
Hmm ok yeah that makes sense then. I wasn't aware that the status could be absent.  If you want to check against an absent status in your original changeset you'll need to compare `result.Status != ""` because go strings can't be nil.